pulsar-io/docs/pom.xml

```diff
@@ -82,6 +82,16 @@
       <artifactId>pulsar-io-debezium-postgres</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
is this related to the pull ?

Reply:
yes. Builds are made in a slightly different way and the maven build must have correct dependencies for `-am` to work.
In the first step, only `core-modules` are built and shared. Other modules are built on demand and that's why dependencies must be correctly specified.